README.md

```diff
@@ -353,6 +353,34 @@ as follows:
 }
 ```
 
+If you are not using NixOS you can place the following flake in ``~/.config/nixpkgs/flake.nix``
```

Comment:
Btw, I think we should replace the `~/.config/nixpkgs/flake.nix` by something more specific, e.g., `~/.config/nixpkgs/home-flake.nix` or something. Seems a bit presumption to steal the generic `flake.nix` file in a directory that we don't even "own" 🙂

Reply:
Created https://github.com/nix-community/home-manager/issues/2466 for this.